{"name": "test_login_with_incorrectly_password", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 404\n +  where 200 = <Response [200]>.status_code", "trace": "self = <API.Tests.test_login.Test_Login object at 0x0000021EA99CA110>\n\n    @allure.description('Login when password incorrectly')\n    def test_login_with_incorrectly_password(self):\n        url = Login_Locators.url_login\n        data = Login_Locators.Invalid_password\n        Req = requests.post(url, json=data)\n>       assert Req.status_code == 404\nE       assert 200 == 404\nE        +  where 200 = <Response [200]>.status_code\n\ntest_login.py:22: AssertionError"}, "description": "Login when password incorrectly", "start": 1675622979124, "stop": 1675622979528, "uuid": "02a1e3be-5cd6-451d-af0b-960da73629f8", "historyId": "024818d21e608740f29497e0a47f306e", "testCaseId": "75232c2ce2bfdf18c58b7f87331c57bf", "fullName": "test_login.Test_Login#test_login_with_incorrectly_password", "labels": [{"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "DESKTOP-4H4LEB1"}, {"name": "thread", "value": "4100-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}